suites/quincy/rados/tier-2_rados_test_bluestore.yaml

```diff
@@ -134,17 +134,18 @@ tests:
         bluestore_cache: true
       desc: Verify tuning of BlueStore cache size for HDDs and SSDs
 
-  - test:
-      name: Bluefs DB utilization
-      desc: DB utilization is under check - bluefs files are not inflated
-      module: test_bluefs_space.py
-      polarion-id: CEPH-83600867
-      config:
-        omap_config:
-          pool_name: re_pool_bluefs_db
-          pg_num: 1
-          pg_num_max: 1
-          obj_start: 0
-          obj_end: 15
-          normal_objs: 400
-          num_keys_obj: 200001
+# commented until CBT bluefs-stats o/p is not verbose
```

Comment:
Make sure you raise a RFE/ bug to have the changes backported to quincy and pacific if needed, and then update the bug in the comment added.